eng/versioning/version_client.txt

```diff
@@ -33,7 +33,7 @@ com.azure:azure-data-appconfiguration;1.1.6;1.2.0-beta.1
 com.azure:azure-data-schemaregistry;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-data-schemaregistry-avro;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-data-tables;12.0.0-beta.2;12.0.0-beta.3
-com.azure:azure-digitaltwins-core;1.0.0-beta.3;1.0.0-beta.4
+com.azure:azure-digitaltwins-core;1.0.0-beta.3;1.0.0
```

Comment:
Unrelated to the changes in this PR, but any reason why we don't have digital twins in the test coverage library pom [here](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/jacoco-test-coverage/pom.xml#L21)

Reply:
I can certainly add this library to that, sure!